Source/WebKit/webpushd/WebPushDaemonMain.mm

```diff
@@ -143,7 +157,13 @@ int WebPushDaemonMain(int argc, char** argv)
             ::WebPushD::WebPushDaemon::singleton().startMockPushService();
         else {
             String libraryPath = NSSearchPathForDirectoriesInDomains(NSLibraryDirectory, NSUserDomainMask, YES)[0];
+
+#if defined(RELOCATABLE_WEBPUSHD)
+            String pushDatabasePath = FileSystem::pathByAppendingComponents(libraryPath, { "WebKit"_s, "WebPush"_s, "PushDatabase.relocatable.db"_s });
```

Comment:
It would be slightly more concise to just have the ASCIILiteral have 2 definitions instead of duplicating the whole line.